app/views/shared/loading/_table.html.erb

```diff
@@ -0,0 +1,24 @@
+<table class="min-w-full table-fixed dark:divide-slate-600">
+  <tbody
+    class="
+      bg-white
+      divide-y
+      divide-slate-200
+      dark:bg-slate-800
+      dark:divide-slate-700
+    "
+  >
+    <% 15.times do %>
```

Comment:
**nitpick:** You have 15 times here (and I think that is what it was previous), but our tables by default load 10 items.

Reply:
Yeah just copied the table loading skeleton from another file. Updated to 10.times in 4fcd5244b945df8c5272cb3be528e6a4f9d19ce2